django_6pm/django_6pm/product/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+
+class Brand(models.Model):
+    brand_name = models.CharField(max_length=250)
+    brand_logo = models.URLField()
+
+
+class Category(models.Model):
+    cat_name = models.CharField(max_length=250)
+    parent_cat_id = models.ForeignKey("self", on_delete=models.SET_NULL, null=True, blank=True)
+
+
+class Product(models.Model):
+    product_id = models.PositiveBigIntegerField(primary_key=True)
+    name = models.CharField(max_length=1000)
+    gender = models.CharField(max_length=100)
```

Comment:
There should be predefined choices for gender field. Learn more about `choices` argument